samples/bluetooth/encrypted_advertising/README.rst

```diff
@@ -0,0 +1,79 @@
+.. _bluetooth_encrypted_advertising_sample:
+
+Bluetooth: Encrypted Advertising
+################################
+
+Overview
+********
+
+This sample demonstrates the usage of the encrypted advertising feature. This
+include:
+
+ - the exchange of the session key and the initialization vector using the Key
+   Material characteristic;
+ - the encryption of advertising payloads;
+ - the decryption of those advertising payloads;
+ - and finally the update of the Randomizer field whenever the RPA is changed.
+
+To use the `bt_ead_encrypt` and `bt_ead_decrypt` functions you need to enable
+the Kconfig symbol :kconfig:option:`CONFIG_BT_EAD`.
+
+This sample uses extended advertising, this is **not** mandatory to use the
+feature. The encrypted adverting feature can be used with legacy advertising as
+well.
+
+Hardware Setup
+**************
+
+This sample use two applications, two devices need to be setup.
+The first one should be flashed with the central and the second one with the
+peripheral.
+
+The two devices should automatically connect based on the device name advertised
+by the peripheral.
```

Comment:
This sentence should be moved to Building and Running so it is in the same paragraph and before the sentence that starts "Here are the outputs".